website/docs/docs-section/deprecated/gsn/gsn-capable-document-store.md

```diff
@@ -24,12 +24,12 @@ You will also need the corresponding address of the [Trust Forwarder](https://do
 To deploy a GsnCapableDocumentStore you can do so by using the following command. You may replace the `<document-store-name>` with a suitable name (the name does not matter).
 
 ```bash
-open-attestation deploy gsn-capable-document-store "My Name" 0x25CEd1955423BA34332Ec1B60154967750a0297D --network ropsten
+open-attestation deploy gsn-capable-document-store "My Name" 0x25CEd1955423BA34332Ec1B60154967750a0297D --network goerli
 ```
 
-> `0x25CEd1955423BA34332Ec1B60154967750a0297D` is the address of trust forwarder for ropsten provided by GSN
+> `0x25CEd1955423BA34332Ec1B60154967750a0297D` is the address of trust forwarder for goerli provided by GSN
 
-This will deploy the GsnCapableDocumentStore on the `ropsten` network. You should see a similar output when the deployment is successful:
+This will deploy the GsnCapableDocumentStore on the `goerli` network. You should see a similar output when the deployment is successful:
```

Comment:
this command works? for goerli netwrok?
do we even need to update this since it has been "deprecated"

Reply:
ok i'll revert it back